src/editors/containers/ProblemEditor/data/OLXParser.js

```diff
@@ -0,0 +1,382 @@
+          answers: element['@_value'].split(' '),
+          feedback: element['#text'],
+        });
+      });
+    }
+    return groupFeedback;
+  }
+
+  parseStringResponse() {
+    const { problem } = this.parsedOLX;
+    const { stringresponse } = problem;
+    const answers = [];
+    let answerFeedback = '';
+    let additionalStringAttributes = {};
+    let data = {};
+    const feedback = this.getFeedback(stringresponse);
+    answers.push({
+      id: indexToLetterMap[answers.length],
+      title: stringresponse['@_answer'],
+      correct: true,
+      feedback,
+    });
+
+    // Parsing additional_answer for string response.
+    const additionalAnswer = _.get(stringresponse, 'additional_answer', []);
+    if (_.isArray(additionalAnswer)) {
+      additionalAnswer.forEach((newAnswer) => {
+        answerFeedback = this.getFeedback(newAnswer);
+        answers.push({
+          id: indexToLetterMap[answers.length],
+          title: newAnswer['@_answer'],
+          correct: true,
+          feedback: answerFeedback,
+        });
+      });
+    } else {
+      answerFeedback = this.getFeedback(additionalAnswer);
+      answers.push({
+        id: indexToLetterMap[answers.length],
+        title: additionalAnswer['@_answer'],
+        correct: true,
+        feedback: answerFeedback,
+      });
+    }
+
+    // Parsing stringequalhint for string response.
+    const stringEqualHint = _.get(stringresponse, 'stringequalhint', []);
+    if (_.isArray(stringEqualHint)) {
+      stringEqualHint.forEach((newAnswer) => {
+        answers.push({
+          id: indexToLetterMap[answers.length],
+          title: newAnswer['@_answer'],
+          correct: false,
+          feedback: newAnswer['#text'],
+        });
+      });
+    } else {
+      answers.push({
+        id: indexToLetterMap[answers.length],
+        title: stringEqualHint['@_answer'],
+        correct: false,
+        feedback: stringEqualHint['#text'],
+      });
+    }
+
+    // TODO: Support multiple types.
+    additionalStringAttributes = {
+      type: _.get(stringresponse, '@_type'),
+      textline: {
+        size: _.get(stringresponse, 'textline.@_size'),
+      },
+    };
+
+    data = {
+      answers,
+      additionalStringAttributes,
+    };
+
+    return data;
+  }
+
+  parseNumericResponse() {
+    const { problem } = this.parsedOLX;
+    const { numericalresponse } = problem;
+    let answers = [];
+    let subAnswers = [];
+    let data = {};
+    if (_.isArray(numericalresponse)) {
+      numericalresponse.forEach((numericalAnswer) => {
+        subAnswers = this.parseResponseObject(numericalAnswer, answers.length);
+        answers = _.concat(answers, subAnswers);
+      });
+    } else {
+      subAnswers = this.parseResponseObject(numericalresponse, answers.length);
+      answers = _.concat(answers, subAnswers);
+    }
+
+    data = {
+      answers,
+    };
+
+    return data;
+  }
+
+  parseResponseObject(numericalresponse, answerOffset) {
```

Comment:
Nit:
Since this is specific to numerical type, maybe name this method `parseNumericResponseObject` to avoid any confusion.